docs/resources/cpts_task.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "Cloud Performance Test Service (CPTS)"
+---
+
+# huaweicloud_cpts_task
+
+Manages a pressure Test Task resource within HuaweiCloud CPTS.
+The task resource only supports serial execution mode.
+
+## Example Usage
+
+```hcl
+resource "huaweicloud_cpts_project" "test" {
+  name = "tf_demo_project"
+}
+
+resource "huaweicloud_cpts_task" "test" {
+  name       = "tf_demo_task"
+  project_id = huaweicloud_cpts_project.test.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the task resource. If omitted, the
+  provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of task, which can contain a maximum of 42 characters.
```

Comment:
Nits：s/the name of task/the name of the task

Reply:
ok